polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -187,7 +152,7 @@ class PythonInterpreter(val symbolTable: RuntimeSymbolTable) extends LanguageInt
 
           jep.set("__polynote_code__", maybeModifiedCode)
           kernelContext.runInterruptible {
-            jep.eval("exec(__polynote_code__, None, __polynote_locals__)\n")
+            jep.eval("exec(__polynote_code__, {**dict(__polynote_globals__), **globals()}, __polynote_locals__)\n")
```

Comment:
Hmm, looks like this syntax is only for python 3.5+, and on travis we have 3.4 (hence the build failures). Not sure whether to get newer python on travis, or try to be more backward-compatible.